### PR TITLE
Add Radio Calor, Sori TV and Impacto TV

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds sone peruvian local channels.

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -175,6 +175,8 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.twitc
 https://iptv.dc-team.com/query?url=https://www.twitch.tv/quatrotvgo
 #EXTINF:-1 tvg-id="Quillavision.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/459651607763457/picture?width=320&height=320" group-title="Local",Quillavision (Cuzco) [Not 24/7]
 http://v4.tustreaming.cl/quillavision/index.m3u8
+#EXTINF:-1 tvg-id="RadioCalorHuancayo.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/RadioCalorHuancayo/picture?width=320&height=320" group-title="Local",Radio Calor (Huancayo) [Not 24/7]
+https://query-streamlink-us.herokuapp.com/iptv-query?streaming-ip=https://www.twitch.tv/calortv
```
[the remark this answers is not in the document]
Comment (Sphinxroot) [reply channels/pe.m3u:181]: not 24/7 stream for now no stream on

[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply channels/pe.m3u:181]: Do not hesitate to add dct's version too : 

https://iptv.dc-team.com/query?url=https://www.twitch.tv/calortv

**Review by freearhey — APPROVED** [review]

File: channels/pe.m3u (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dum4G) [reply channels/pe.m3u:181]:
```suggestion
https://query-streamlink-us.herokuapp.com/iptv-query?streaming-ip=https://www.twitch.tv/calortv
#EXTINF:-1 tvg-id="RadioCalorHuancayo.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/RadioCalorHuancayo/picture?width=320&height=320" group-title="Local",Radio Calor (Huancayo) [Not 24/7]
https://iptv.dc-team.com/query?url=https://www.twitch.tv/calortv
```

[the remark this answers is not in the document]
Comment (sguinetti) [reply channels/pe.m3u:181]: Fixed. Thank you for this suggestion.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**